test/parallel/test-dgram-bind-shared-ports.js

```diff
@@ -64,12 +64,15 @@ if (cluster.isMaster) {
 
   socket1.bind({
     address: 'localhost',
-    port: common.PORT,
+    port: 0,
     exclusive: false
-  }, () => {
-    socket2.bind({ port: common.PORT + 1, exclusive: true }, () => {
+  }, common.mustCall(() => {
+    socket2.bind({
+      port: socket1.address().port + 1,
```

Comment:
To be clear, the change already in place in this test is incorrect because it guarantees that socket1 doesn't use the same port in worker1 and worker2.

I think the way to go is to move the test as it currently exists on the master branch from the `parallel` directory to the `sequential` directory.